### ISSUE TITLE
PT-BR spell file

### ISSUE THREAD

**Issue by mMontu** [vim/vim#1330]
The pt.utf-8.spl file downloaded by spellfile.vim for Brazilian Portuguese displays regular words as wrong when they are joined with dash or dots. For example, the following is displayed as correct:

```
Teste arquivo
Teste,arquivo

```
, while these are marked wrong:

```
Teste.arquivo
Teste-arquivo
```

I didn't understand what caused the problem in the first place. 
But it can be solved generating a new .spl file by following the recipe in the runtime/spell directory, using the spell file from [extensions.openoffice.org](http://extensions.openoffice.org/en/project/vero-brazilian-portuguese-spellchecking-dictionary-hyphenator)  and applying [this patch](https://github.com/vim/vim/files/655099/pt_BR.diff.zip) instead of `runtime/spell/pt/pt_BR.diff`.

**Comment by brammool** [comment]
Marcelo D Montu wrote:

> The pt.utf-8.spl file downloaded by spellfile.vim for Brazilian
> Portuguese displays regular words as wrong when they are joined with
> dash or dots. For example, the following is displayed as correct:
> 
> ```
> Teste arquivo
> Teste,arquivo
> 
> ```
> , while these are marked wrong:
> 
> 
> ```
> Teste.arquivo
> Teste-arquivo
> ```
> 
> I didn't understand what caused the problem in the first place. 
> But it can be solved generating a new .spl file by following the recipe in the runtime/spell directory, using the spell file from [extensions.openoffice.org](http://extensions.openoffice.org/en/project/vero-brazilian-portuguese-spellchecking-dictionary-hyphenator)  and applying [this patch](https://github.com/vim/vim/files/655099/pt_BR.diff.zip) instead of `runtime/spell/pt/pt_BR.diff`.


Thanks.  Perhaps you can make the resulting spell file available, so
that others can try it out and check that it works well.


-- 
hundred-and-one symptoms of being an internet addict:
126. You brag to all of your friends about your date Saturday night...but
     you don't tell them it was only in a chat room.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by mMontu** [comment]
Of course. It can [be downloaded here](https://github.com/vim/vim/files/657554/pt.utf-8.spl.zip)
